app/ui/browser/reducers/ui-state.js

```diff
@@ -23,22 +25,31 @@ export default function uiState(state = initialState, action) {
       return state.set('statusText', action.text);
 
     case types.LOCATION_CHANGED:
+      assert(isUUID(action.pageId), 'LOCATION_CHANGED requires a page id.');
```

Comment:
I had been meaning to ask about our use of assert, so created an issue for discussion of it, #550